ArduPlane/Plane.h

```diff
@@ -789,6 +789,9 @@ class Plane : public AP_Vehicle {
     bool non_auto_terrain_disable;
     bool terrain_disabled();
 
+    // air mode state: off=0, on=1
```

Comment:
This is now an enumeration; shouldn't be hinting at numbers here.

Reply:
done